res/blackbox-tests/configuration.toml

```diff
@@ -69,6 +69,7 @@
 BootTimeout = "30s"
 CheckInterval = "10s"
 Host = "localhost"
+ServerBindAddr = ""
```

Comment:
Add comment why left blank. See example in the issue. https://github.com/edgexfoundry/app-service-configurable/issues/93